package.json

```diff
@@ -51,42 +51,42 @@
 		"type-check": "tsc --noEmit"
 	},
 	"dependencies": {
-		"@typescript-eslint/utils": "^5.58.0"
+		"@typescript-eslint/utils": "^5.62.0"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.21.4",
-		"@babel/eslint-parser": "^7.21.3",
-		"@babel/eslint-plugin": "^7.19.1",
-		"@commitlint/cli": "^17.5.1",
-		"@commitlint/config-conventional": "^17.4.4",
-		"@swc/core": "^1.3.75",
-		"@swc/jest": "^0.2.28",
+		"@babel/core": "^7.25.8",
+		"@babel/eslint-parser": "^7.25.8",
+		"@babel/eslint-plugin": "^7.25.7",
+		"@commitlint/cli": "^17.8.1",
+		"@commitlint/config-conventional": "^17.8.1",
+		"@swc/core": "^1.7.36",
+		"@swc/jest": "^0.2.36",
 		"@types/jest": "^27.5.2",
-		"@types/node": "^16.18.23",
-		"@typescript-eslint/eslint-plugin": "^5.58.0",
-		"@typescript-eslint/parser": "^5.58.0",
-		"del-cli": "^5.0.0",
-		"eslint": "^8.38.0",
-		"eslint-config-prettier": "^8.8.0",
-		"eslint-doc-generator": "^1.4.3",
-		"eslint-import-resolver-typescript": "^3.6.1",
-		"eslint-plugin-import": "^2.29.0",
-		"eslint-plugin-jest": "^27.2.1",
+		"@types/node": "^16.18.113",
```

Comment:
This one should actually be v20 since we changed the version in `nvmrc`. Could you change that here?